src/com/maddyhome/idea/vim/group/ChangeGroup.java

```diff
@@ -1910,6 +1910,56 @@ private void exitAllSingleCommandInsertModes(@NotNull Editor editor) {
     }
   }
 
+  /**
+   * Deletes next search, or current if cursor is on a match.
+   *
+   * @param editor  The editor to insert the text into
+   * @param caret
+   * @param context The data context
+   * @param count   The number times to search before deleting
+   * @param forwards
+   * @return true if able to find and delete the search, false if not
+   */
+  public boolean deleteNextSearch(Editor editor, Caret caret, DataContext context, int count, boolean forwards) {
+    String lastSearch = VimPlugin.getSearch().getLastSearch();
+    int currentPos = editor.getCaretModel().getOffset();
+    TextRange nextRange = new TextRange(currentPos,currentPos);
+    for(int i = 0; i< count; i++) {
+      nextRange = VimPlugin.getSearch().findCurrentOrNext(editor, lastSearch, nextRange.getStartOffset(), false, forwards);
```

Comment:
`findCurrentOrNext ` is a static method. It can be called like `ChangeGroup.findCurrentOrNext(...)` isn't it?

Reply:
Yes, changed